plugins/src/main/java/com/google/fhir/proxy/plugin/OpenSRPSyncAccessDecision.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2021-2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.fhir.proxy.plugin;
+
+import ca.uhn.fhir.rest.api.RequestTypeEnum;
+import ca.uhn.fhir.rest.server.servlet.ServletRequestDetails;
+import com.google.fhir.proxy.interfaces.AccessDecision;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+import org.apache.http.HttpResponse;
+import org.apache.http.util.TextUtils;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class OpenSRPSyncAccessDecision implements AccessDecision {
+
+	public static final String CARE_TEAM_TAG_URL = "http://smartregister.org/fhir/care-team-tag";
+
+	public static final String LOCATION_TAG_URL = "http://smartregister.org/fhir/location-id";
```

Comment:
Can we move these urls to the configuration file?

Reply:
Hey Reham, which file are we using as the configuration file currently? I probably don't have this file locally?

Reply:
HI @ekigamba , 
The proxy design does not have any configuration file. Rather, they work on reading configurations data from the environment variables.